index.js

```diff
@@ -15,7 +15,39 @@ function promisifyOwnProperties(obj) {
     .reduce((acc, curr) => Object.assign(acc, curr), {});
 }
 
+function handleArray(arr) {
+  let numberOfPromisifictions = 0;
+
+  const promisifictions = arr.map(e => {
```

Comment:
this isn't quite what i intended; i'm not saying it should promisify an array of functions.

I'm expecting `const a = [1]; a.foo = function (callback) {};` to promisify to `[1]` with a "foo" property that's a promisified function.